avm/utilities/pipelines/platform/Get-AvmCsvData.ps1

```diff
@@ -0,0 +1,61 @@
+<#
+.SYNOPSIS
+Parses AVM module CSV file
+
+.DESCRIPTION
+Depending on the parameter, the correct CSV file will be parsed and returned a an object
+
+.PARAMETER ModuleIndex
+Type of CSV file, that should be parsed ('Bicep-Resource', 'Bicep-Pattern')
```

Comment:
```suggestion
Type of CSV file, that should be parsed. E.g., 'Bicep-Resource'
```

When using PowerShells Get-Help, the allowedSet will be displayed anyways. No need to duplicate. I'd at most add an example.